Formula/lighthouse.rb

```diff
@@ -0,0 +1,35 @@
+class Lighthouse < Formula
+  desc "Rust Ethereum 2.0 Client"
+  homepage "https://github.com/sigp/lighthouse"
+
+  url "https://github.com/sigp/lighthouse.git",
+      tag:      "v1.4.0",
+      revision: "3b600acdc5bf9726367c18277a22486573b8b457"
```

Comment:
Do we need the Git clone? Can we use a tarball instead?

Reply:
Good question, I'll check and update the formula if that works.

Reply:
yeah, I dont think we need, there is a empty `.gitmodules` file